config/initializers/paperclip.rb

```diff
@@ -0,0 +1,2 @@
+Paperclip::Attachment.default_options[:url] = ':s3_domain_url'
+Paperclip::Attachment.default_options[:path] = '/:class/:attachment/:id_partition/:style/:filename'
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.